ticketutil/servicenow.py

```diff
@@ -0,0 +1,459 @@
+import os
+import logging
+import requests
+import json
+
+from ticketutil.ticket import Ticket
+
+__author__ = 'dranck, rnester, kshirsal, pzubaty'
+
+STATE = {'new':'0',
```

Comment:
Is this mapping just based off our internal SN instance, or is this consistent across all instances? Reason I'm asking is because for Redmine, I had to query the API to see what status ID corresponded to a status name. If SN allows for custom states, wouldn't the state's ID have to be retrieved somehow?

See https://github.com/dmranck/ticketutil/blob/master/ticketutil/redmine.py#L368 for Redmine _get_status_id() helper function.